docs/source/man.rst

```diff
@@ -306,8 +306,8 @@ The following options are available for every Quipucords command.
 Examples
 --------
 
-:Creating a new credential with a keyfile: ``qpc cred add --name=new-creds --username=qpc-user --sshkeyfile=/etc/ssh/ssh_host_rsa_key``
-:Creating a new credential with a password: ``qpc cred add --name=other-creds --username=qpc-user-pass --password``
+:Creating a new credential with a keyfile: ``qpc cred add --name=new-creds **--type=** *network* --username=qpc-user --sshkeyfile=/etc/ssh/ssh_host_rsa_key``
```

Comment:
maybe change to say "Creating a new network type credential ...". And also provide an example of creating a vcenter example?